media/stylus/components/structure.styl

```diff
@@ -54,6 +54,16 @@ draw-grid();
 
 #tabzilla {
     bidi-value(float, right, left);
+    display: block;
+    height: 44px;
+    width: 150px;
+    overflow: hidden;
+}
+
+#tabzilla-panel {
+    /* in case the javascript loads before the CSS */
+    height: 0px;
```

Comment:
Our compressor probably takes care of this, but just to play it safe, maybe `0` instead of `0px`?